app/static/js/userManagement.js

```diff
@@ -63,6 +63,20 @@ function submitRequest(method, username){
     }
   })
 }
+export function populateModal(programName, programDescription, partner, contactEmail, contactName, location, programId, instagramUrl, facebookUrl, bereaUrl) {
+  document.getElementById('programName').value = programName;
+  document.getElementById('programDescription').value = programDescription;
+  handleFileSelection('modalProgramImage')
+  document.getElementById('partner').value = partner;
+  document.getElementById('contactEmail').value = contactEmail;
+  document.getElementById('contactName').value = contactName;
+  document.getElementById('location').value = location;
+  document.getElementById('instagramUrl').value = instagramUrl;
+  document.getElementById('facebookUrl').value = facebookUrl;
+  document.getElementById('bereaUrl').value = bereaUrl;
```

Comment:
We should use jquery like we do everywhere else

Reply:
`$("#programName").val(programName)`

Reply:
We used jQuery.